code/modules/projectiles/guns/projectile/automatic/mac.dm

```diff
@@ -49,7 +49,7 @@
 	update_icon()
 
 ///not for typical use: it WILL cause lag
-/obj/item/gun/projectile/automatic/mac/croon
+/* /obj/item/gun/projectile/automatic/mac/croon
```

Comment:
This is causing a compile error. code/game/objects/random/guns.dm, line 122 is calling for this obj but you've commented it out. Please resolve by either removing the comment here or removing the call in the above file.